config/manifests/bases/managed-fusion-agent.clusterserviceversion.yaml

```diff
@@ -11,7 +11,14 @@ metadata:
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
-  customresourcedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: ManagedFusionOffering is the Schema for the managedfusionofferings
+        API
+      displayName: Managed Fusion Offering
+      kind: ManagedFusionOffering
+      name: managedfusionofferings.misf.ibm.com
+      version: v1alpha1
```

Comment:
are we sure that this can't be generated via kustomize or any kubebuilder tags?

Reply:
It can be generated via the make manifests command.